oci8.go

```diff
@@ -245,6 +245,10 @@ func (drv *DriverStruct) Open(dsnString string) (driver.Conn, error) {
 				C.OCIHandleFree(unsafe.Pointer(conn.errHandle), C.OCI_HTYPE_ERROR)
 				conn.errHandle = nil
 			}
+			if conn.txHandle != nil {
```

Comment:
Could you  move this above userSession?
I know it is kind of silly but it is in reverse order of creation, just helps keeping track of everything.

Reply:
Will do!